Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -220,29 +220,5 @@ void ISerializationCallbackReceiver.OnAfterDeserialize()
         }
 
         #endregion  ISerializationCallbackReceiver Implementation
-
-        #region Mixed Reality Controller Mapping helpers
-
-        // TODO - needs validation to ensure duplicates are not added?
-
-        public MixedRealityControllerMapping GetControllerMapping(Type controllerType, Handedness hand)
```

Comment:
This is addressed in the next PR.

Reply:
Will be moved to base controller in devices refactor. @SimonDarksideJ do you want me to revert until then?